dacapo/experiments/tasks/hot_distance_task_config.py

```diff
@@ -8,14 +8,23 @@
 
 @attr.s
 class HotDistanceTaskConfig(TaskConfig):
-    """This is a Hot Distance task config used for generating and
-    evaluating signed distance transforms as a way of generating
-    segmentations.
-
-    The advantage of generating distance transforms over regular
-    affinities is you can get a denser signal, i.e. 1 misclassified
-    pixel in an affinity prediction could merge 2 otherwise very
-    distinct objects, this cannot happen with distances.
+    """Class for generating and evaluating signed distance transforms as a way of generating
```

Comment:
This seems like an inaccurate description for the TaskConfig (it isn't generating anything, right?)

Reply:
@mzouink